requirements-dev.txt

```diff
@@ -9,4 +9,4 @@
 --requirement requirements-dev-sqlalchemy.txt
 --requirement requirements-dev-test.txt
 --requirement requirements-dev-build.txt
---requirement requirements-dev-publish.txt
+--requirement requirements-dev-publish.txt
```

Comment:
This is strange, I don't see a change here. Would you mind terribly removing this to avoid confusion later?